packages/manager/tools/component-rollup-config/src/index.ts

```diff
@@ -31,94 +32,124 @@ const getLanguages = (pluginsOpts) => {
   return get(pluginsOpts, 'translations.languages');
 };
 
-const generateConfig = (opts, pluginsOpts) => mergeConfig({
-  plugins: [
-    peerdeps(),
-    html(),
-    json({
-      preferConst: true,
-      compact: true,
-      namedExports: false,
-    }),
-    lessTildeImporter(pluginsOpts.lessTildeImporter),
-    lessInject({
-      option: {
-        plugins: [
-          lessPluginRemcalc,
-        ],
-      },
-    }),
-    sass({
-      insert: true,
-      output: false,
-    }),
-    image({
-      output: `./dist/assets/${defaultName}`,
-    }),
-    resolve(),
-    commonjs(),
-    translationInject({
-      languages: getLanguages(pluginsOpts),
-    }),
-    translationUiRouter({
-      subdirectory: 'translations',
-      languages: getLanguages(pluginsOpts),
-    }),
-    dynamicImportVars(),
-    babel({
-      babelHelpers: 'bundled',
-      babelrc: false,
-      exclude: 'node_modules/**',
+const generateConfig = (opts, pluginsOpts) =>
+  mergeConfig(
+    {
       plugins: [
-        '@babel/plugin-proposal-class-properties',
-        '@babel/plugin-proposal-optional-chaining',
-        '@babel/plugin-proposal-private-methods',
-        '@babel/plugin-syntax-dynamic-import',
-        'babel-plugin-angularjs-annotate',
-      ],
-      presets: [
-        ['@babel/preset-env'],
+        peerdeps(),
+        html(),
+        json({
+          preferConst: true,
+          compact: true,
+          namedExports: false,
+        }),
+        lessTildeImporter(pluginsOpts.lessTildeImporter),
+        lessInject({
+          option: {
+            plugins: [lessPluginRemcalc],
+          },
+        }),
+        sass({
+          insert: true,
+          output: false,
+        }),
+        image({
+          output: `./dist/assets/${defaultName}`,
+        }),
+        resolve(),
+        commonjs(),
+        translationInject({
+          languages: getLanguages(pluginsOpts),
+        }),
+        translationUiRouter({
+          subdirectory: 'translations',
+          languages: getLanguages(pluginsOpts),
+        }),
+        dynamicImportVars({
+          exclude: /node_modules\//,
```

Comment:
**Note:** here is the only part that has changed (do not pay attention to the linting applied).